config/packages.cmake

```diff
@@ -231,6 +231,8 @@ if(ENABLE_PARTITIONING)
 
 endif()
 
+cinch_target_link_libraries(flecsi "${FLECSI_RUNTIME_LIBRARIES};${PARTITION_LIBRARIES};${IO_LIBRARIES}")
```

Comment:
Is that how you have to pass it.  I was thinking it was
```
cinch_target_link_libraries(flecsi "${FLECSI_RUNTIME_LIBRARIES} ${PARTITION_LIBRARIES} ${IO_LIBRARIES}")
```
without the semi-colons.